### ISSUE TITLE
Feature request: Select multiple tags for search

### ISSUE THREAD

**Issue by advert665** [pjeby/tag-wrangler#119]
I really like this plugin, the only thing I miss from the vanilla version is the ability to hold control and select multiple tags for a search (in an ideal world I'd love to be able to alt click and exclude too). I don't use tag pages. Do you think it would be possible to add a menu to choose shift/control/alt + click functionality in the tag pane? Thanks!

**Comment by pjeby** [comment]
Do note that you can right-click and choose to add or remove tags or their exclusions via the popup menu.

But yes, I do plan to add some settings about it; I'm mostly stalled on design because I don't want to have to implement it twice, but it's not something I personally care about, and I'm not entirely sure how I want to prioritize the options.   Further complicating things is #114, where some of those modifiers are also used by the Tag Folder plugin, or the possibility some people might want different functions for tags clicked in different places.

Personally, if I wanted to frequently use tag search control, I'd add some buttons to the UI to toggle include/exclude/don't-care, rather than using modifier keys or the menu options.  Or I'd use TagFolder or another similar plugin.  Currently I use it infrequently enough that menu options are fine for me personally.

My current thought is leaning towards having something like "Clicking tags opens tag pages:" with dropdown choices of Always, Never, When Tag Page Exists, and When Alt/Opt Is Down.  I think for most people who don't like the current behavior, one of those options (e.g. Never) will likely suffice.

**Comment by greetclammy** [comment]
>My current thought is leaning towards having something like "Clicking tags opens tag pages:" with dropdown choices of Always, Never, When Tag Page Exists, and When Alt/Opt Is Down. I think for most people who don't like the current behavior, one of those options (e.g. Never) will likely suffice.

Yes please! I would select "Never" to have the vanilla Ctrl/Cmd click on tag behavior back! (Toggle it in search query.) I have no need for tag pages.
